Agreements/Internal/TCCPT_TCSW.txt

```diff
@@ -0,0 +1 @@
+COPX:*:*:CPT:EGLF:*:London TC CPT:London TC SW:*:16000:CPT
```

Comment:
We don't currently have this agreement - all individually coordinated. Presumably the reason there isn't a permanent agreement is because there is too much conflict here that a suitable level needs to be determined each time? I think I'd rather leave these without an agreement for now and we can discuss later